src/frontend/src/app/scheduler/create-schedule/create-schedule.component.ts

```diff
@@ -1,19 +1,16 @@
 import { Component, OnInit, EventEmitter, Input, Output } from '@angular/core';
 import { SchedulesService, AlertService } from '../../services/index';
+import { FormBuilder, FormGroup, Validators, FormControl } from '@angular/forms';
 import Utils from '../../utils';
+import { CustomValidator } from '../../directives/custom-validator';
```

Comment:
is this generic validator and can be used elsewhere too; or specific to CreateSchedule?

Reply:
This is generic and can be used anywhere per requirement.